src/main/java/com/icthh/xm/tmf/ms/activation/service/SagaTaskExecutor.java

```diff
@@ -25,6 +25,12 @@
 
     @LogicExtensionPoint(value = "OnFinish", resolver = TransactionTypeKeyResolver.class)
     public void onFinish(SagaTransaction sagaTransaction) {
-        log.warn("Script for finish not found. Transaction {}.", sagaTransaction);
+        log.info("Script for finish not found. Transaction {}.", sagaTransaction);
+    }
+
+    @LogicExtensionPoint(value = "TaskCheckWaitCondition", resolver = TaskTypeKeyResolver.class)
```

Comment:
propose rename `TaskCheckWaitCondition` to `OnCheckWaitCondition`

Reply:
done